### PR TITLE
Update readme with "how to use"

### PR DESCRIPTION
**Pull request by chieftinkerer**

### CODE REVIEW

File: README.md
Comment (XVincentX) [inline README.md]: This is not correct. Once published the image would be just `express-gateway`

**Review by XVincentX — CHANGES_REQUESTED** [review]
Thanks a lot for the PR — I'm ashamed I didn't notice this earlier.

This readme is useful, although the real readme is in the "official-images" repository on the Docker Organization.

I've noted an error — which hopefully you can easily fix.

Thanks a lot for that!

File: README.md
Comment (chieftinkerer) [reply README.md]: Hi XVincentX.  I think you are suggesting to remove the ":latest" tag from the image name on the docker pull instructions.  That is fine, it defaults to "latest" if no tag is provided. I tend to prefer explicit tags which is why I added it.  I will make that change and resubmit.  Thanks.

**Comment by chieftinkerer** [comment]
Updated.  Let me know if you have suggested changes.  Thanks.

File: README.md
Comment (XVincentX) [reply README.md]: Hmm no, that was not my intention.

Let me clarify: the current Docker Image we're publishing on [DockerHub](https://hub.docker.com/r/expressgateway/express-gateway/) is not generated using this file, but [another one](https://github.com/ExpressGateway/express-gateway/blob/master/Dockerfile).

This repository is to incorporate Express Gateway as an [official Docker Image](https://github.com/docker-library/official-images/pull/3980) — which currently is still waiting to be merged. When that happens, the official image name will simply be `express-gateway`.

File: README.md
Comment (XVincentX) [inline README.md]: Again, this must be `docker pull express-gateway` — when the official PR will be merged.

**Review by XVincentX — CHANGES_REQUESTED** [review]
I appreciate a lot this readme and it's going to be a great starting point for our official https://express-gateway.io home page for installation methods.

**Comment by chieftinkerer** [comment]
I will update the docker hub page(s) if you give me access.  The current readme up there talks about installation of the gateway as a global app - so, not specific to the docker image which I think people will want to see on the hub page.  We can also add links to the dockerfile that was used to create the docker image(s) they are pulling.

**Comment by XVincentX** [comment]
The current DockerHub readme is generated automatically from the Express Gateway's repository README, and unfortunately there's no way to change that. We really need to wait for people in DockerHub to review [the PR](https://github.com/docker-library/official-images/pull/3980/files).

File: README.md
Comment (chieftinkerer) [reply README.md]: Not sure that I am following.  The image up there today requires that you provide the docker repository - e.g. expressgateway/<image name>.  A pull of an "official" image would work after you have an "offical" repository.  But, all I see is a public repo that you have created, so still needs the repo name in the pull.

[@localhost docker-express-gateway]$ docker images | grep express-gateway
**expressgateway**/express-gateway                                   latest              41604254e8c9        12 days ago         185MB
[j@localhost docker-express-gateway]$ docker pull express-gateway
Using default tag: latest
Error response from daemon: pull access denied for express-gateway, repository does not exist or may require 'docker login'
[j@localhost docker-express-gateway]$ 

Is there an official docker hub repo that I am not seeing?  Thanks.

Comment (XVincentX) [reply README.md]: No — we do not have an official docker image yet, and that's the whole point. This repository is meant to store the official docker file definition, and it should not be used by anybody yet.

Does that clarify?

Comment (chieftinkerer) [reply README.md]: That explains why you are suggesting that change.  However, it is out there and public.  So, if you tell people to pull without the repo name it will fail.  Can I suggest that you make the current repo private until you are ready to make official ... or I will make the change you want and add a note explaining it ... somewhere ... that can be removed when the repo is granted official status.  I would be concerned with publishing instructions in a public hub repo that will not currently work.   Thoughts?

Comment (XVincentX) [reply README.md]: > So, if you tell people to pull without the repo name it will fail. 

I'm not suggesting that — the current `README` it's not stating any installation process at all (because we're missing the last step)

> Can I suggest that you make the current repo private until you are ready to make official

Unfortunately I can't — it is required to leave it public so the automatic scripts on `docker-library` can be triggered for verification.

My plan was to approve the PR and defer the merge when this will be finally finalized. Does that sound good?

Comment (chieftinkerer) [reply README.md]: Sounds good.  Making the change.  Thanks for the clarity on how you are planning to roll this out.

**Review by XVincentX — APPROVED** [review]

**Comment by XVincentX** [comment]
That is perfect. Once we have guys in Docker Library merging that, this will be merged too.

We can go there and make some loud to make the process faster :trollface:

**Comment by XVincentX** [comment]
We finally have the official docker image.

**Pull request merged**
